Tests/Realm.Tests/Database/RealmDictionaryTests.cs

```diff
@@ -1073,6 +1073,56 @@ public void ObjectFromAnotherRealm_ThrowsRealmException()
             Assert.That(() => realm2.Write(() => obj2.EmbeddedObjectDictionary["foo"] = embeddedItem), Throws.TypeOf<RealmException>().And.Message.Contains("embedded object that is already managed"));
         }
 
+        [Test]
+        public void AsRealmQueryable_RaisesNotifications()
+        {
+            var obj = _realm.Write(() =>
+            {
+                return _realm.Add(new DictionariesObject());
+            });
+
+            var dogToAge = obj.ObjectDictionary;
```

Comment:
It's a very small thing, and I understand why you did it, but it looks a little strange to see variables called `dogToAge` and `oldDogs` when there's not property called like that 😁 .
I think it would make sense to have more appropriate names